gui/mozregui/mainwindow.py

```diff
@@ -75,6 +76,13 @@ def __init__(self):
         self.ui.log_view.setVisible(True)
         self.ui.logDockWidget.setVisible(True)
 
+        # Use better toolbar icons in dark mode
+        dark_mode_enabled = QGuiApplication.palette().color(QPalette.Window).lightness() < 128
```

Comment:
This code is repeated in `gui/mozregui/report.py`, `gui/mozregui/report_delegate.py`, and `gui/mozregui/skip_chooser.py`. Perhaps it can be moved somewhere where it can be reused.